deploy/11-stake-metafarm/index.ts

```diff
@@ -0,0 +1,66 @@
+import { DeployerFn } from "@ubeswap/hardhat-celo"
+import {
+  MoolaStakingRewards__factory,
+  ERC20__factory
+} from "../../typechain"
+import { ContractTransaction } from "ethers"
+import { ethers } from "ethers"
+
+const STAKING_TOKEN_ADDRESS = '0x25938830fbd7619bf6cfcfdf5c37a22ab15a93ca' // mcUSD / mcUSD-mcEUR-FP liquidity pool
+
+const MOOLA_STAKING_REWARDS_ADDRESS = '0x26B819D77CcaB96253F5756760EFE3D57dCccf14'
+
+const AMOUNT_STAKING_TOKEN = ethers.BigNumber.from("1000000000000000") // .001 mcUSD / mcUSD-mcEUR-FP
+
+function sleep(ms: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+
+export const doTx = async (
```

Comment:
this same function appears to be defined in a few different places, could it be imported from somewhere instead?